apps/wing/src/cli.ts

```diff
@@ -191,6 +191,11 @@ async function main() {
       "-o, --output-file <outputFile>",
       "File name to write test results to (file extension is required, supports only .json at the moment)"
     )
+    .addOption(
+      new Option("-R, --retry [retries]", "Number of times to retry failed tests")
+        .preset(3)
```

Comment:
Not sure if we want this as a default. During development iteration, test failures will be normal and retying them will just lead to noise.

Reply:
What do you mean "as a default"? That's only if you use the `--retry` option in the command.

Reply:
I could probably add an extra test for when the `--retry` parameter isn't passed, to check that it actually doesn't retry failed tests if you want.

Reply:
If you want, I could probably add an extra test for when the --retry parameter isn't passed, to check that it actually doesn't retry failed tests.

Reply:
If you want, I could probably add an extra test for when the `--retry` parameter isn't passed, to check that it actually doesn't retry failed tests.

Reply:
To sum it up:
- `wing test [...endpoint]` = 0 retries
- `wing test [...endpoint] --retry` = 3 retries
- `wing test [...endpoint] --retry x` = x retries

Reply:
Gotcha, I misread it, yeah then that makes sense

Reply:
For the sake of completeness, I've added a test for when the option is missing.